app/views/reviews/_review.html.erb

```diff
@@ -1 +1,14 @@
-<%# Review card for the experiences#show %>
+<div class="col col-6">
+  <div class="host-container p-3 mt-3">
+    <div class="host-name-container">
+      <%= cl_image_tag review.user.photo.key, class: "avatar-large" %>
```

Comment:
I think this doesn't work on the latest main, but you can use the new avatar component that has been added:
```suggestion
      <%= render 'avatar', user: review.user, large: true %>
```